src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/sendtofirsttier/SendToFirstTierSubmittedHandler.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.sendtofirsttier;
+
+import static java.util.Objects.requireNonNull;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.EventType;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
+import uk.gov.hmcts.reform.sscs.ccd.service.CcdCallbackMapService;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class SendToFirstTierSubmittedHandler implements PreSubmitCallbackHandler<SscsCaseData> {
+
+    private final CcdCallbackMapService ccdCallbackMapService;
+
+    @Value("${feature.postHearingsB.enabled}")
+    private final boolean isPostHearingsBEnabled;
+
+    @Override
+    public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callback) {
+        requireNonNull(callback, "callback must not be null");
+        requireNonNull(callbackType, "callbacktype must not be null");
+
+        return callbackType.equals(CallbackType.SUBMITTED)
+                && callback.getEvent() == EventType.SEND_TO_FIRST_TIER
+                && isPostHearingsBEnabled;
+    }
+
+    @Override
+    public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType, Callback<SscsCaseData> callback,
+                                                          String userAuthorisation) {
```

Comment:
need to call canHandle() at the start of the method, check other handlers to see the generic way of doing it

Reply:
Done